components/Editor/index.tsx

```diff
@@ -82,49 +93,94 @@ const Editor = ({ readOnly = false }: Props) => {
   const [callValue, setCallValue] = useState('')
   const [unit, setUnit] = useState(ValueUnit.Wei as string)
 
-  const handleWorkerMessage = (event: MessageEvent) => {
-    const { code: byteCode, warning, error } = event.data
+  const log = useCallback(
+    (line: string, type = 'info') => {
+      // See https://blog.logrocket.com/a-guide-to-usestate-in-react-ecb9952e406c/
```

Comment:
lets remove this comment

Reply:
I would rather keep it, it explains why the code looks weird :-)
And this is something that's rather unexpected, so it's good to keep the link

Reply:
We agreed to keep it